online-store/src/components/data/data.ts

```diff
@@ -0,0 +1,225 @@
+export interface IItem {
+  id: number;
+  name: string;
+  quantity: number;
+  year: number;
+  manufacturer: string;
+  color: string;
+  cams: number;
+  favorite: boolean;
+}
+
+export const data: IItem[] = [
```

Comment:
This data potentially comes from backed, so I would rather put it into json file or even use something like this https://jsonplaceholder.typicode.com/